File: scripts/MATLAB/functions&packages/RREX_Functions/Ivane_toolbox/ganachaud/Ml/routines/ffread_check.m
   a=['always'];
   b=['looking sky'];
   ii=[ -1 2 3 4 5 6 8 7];
   jj=[ -1 2 -3 4 -5 6 32768 -32769];
   kk=[ -1 2 -3 4 -5 6 -7 2147483647 -2147483648 ];
   x=[ 1 2 3 4 5 6 7 8 9 ];
   y=[ 9 8 7 6 5 4 3 2 1 ];
   clear

   fid=fopen('temp.bin','r');
   disp(setstr(ffread(fid,inf,'char'))')
   disp(setstr(ffread(fid,11,'char'))')
   disp(ffread(fid,inf,'short'))
   disp(ffread(fid,inf,'int'))
   disp(ffread(fid,inf,'long'))
   disp(ffread(fid,inf,'float'))
   disp(ffread(fid,[2,9],'double'))
   fclose(fid);

File: scripts/MATLAB/functions&packages/RREX_Functions/Ivane_toolbox/ganachaud/Ml/routines/ffread_check.f
      character a*6,b*11
      integer*2   ii(8)
      integer     jj(8)
      integer     kk(9)
      real        x(9)
      real*8        y(2,9)
      do i=1,8
         ii(i)=i
         jj(i)=-i
      enddo
      do j = 1,9
         kk(j)=2*j
         x(j)=100.*j
         y(1,j)=j
         y(2,j)=-j
      enddo
      a="always"
      b="looking sky"
      open(10,file='temp.bin',form='unformatted',status='write')
      write(10) a
      write(10) b
      write(10) ii
      write(10) jj
      write(10) kk
      write(10) x
      write(10) y

      print*, a 
      print*, b
      print*, ii
      print*, jj
      print*, kk
      print*, x 
      print*, y
      close(10)
      end
